com.civicrm.hrjobroles/js/src/job-roles/services/hr-job-roles-service.js

```diff
@@ -417,15 +417,29 @@ define([
 
             },
 
-            getNewJobRole: function getNewJobRole(contract_id) {
+            getNewJobRole: function (contract_id) {
                 //Creates new JobRole depending on contract id and returns promise
                 return CRM.api3('HrJobRoles', 'create', {
                     "sequential": 1,
                     "job_contract_id": contract_id,
                     "title": ''
                 });
-            }
+            },
 
+            /**
+             * [getCurrentDepartments description]
+             * @param  {[type]} contractId [description]
+             * @return {[type]}            [description]
+             */
+            getCurrentDepartments: function (contractId) {
```

Comment:
Its not clear to me that why we are using 2 then statement? 
1. Here
2. Where it is called i.e. hr-job-roles-controller.js